### ISSUE TITLE
[BUG] API Export with per method override should use operation name from Swagger instead of id from API Manager

### ISSUE THREAD

**Issue by shan916** [Axway-API-Management-Plus/apimanager-swagger-promote#173]
**Swagger-Promote Version**
1.6.3

**API-Manager and Service-Pack Version**
7.5.3 SP7

**Expected behavior**
When exporting an API that has per method overrides for Inbound Security, the exported **api-config.json** should use the `operationId` from the Swagger definition.

**Actual behavior**
The exported **api-config.json** uses the id of the operation instead of the `operationId` from the Swagger definition.

**Test case sample**
Create an API in API Manager with per method overrides for Inbound Security. Then, run the API Export script.

The generated **api-config.json** uses the id of the operation from API Manager.

```
{
  "name" : "petstore-api",
  "path" : "/petstore-api/pet",
  "state" : "unpublished",
  "version" : "1.0.0",
  "organization" : "Test Org",
  "apiDefinition" : "petstore-api.json",
  "summary" : "Test automation with the Pet Store API",
  "vhost" : "api-test.example.com",
  "inboundProfiles" : {
    "_default" : {
      "monitorAPI" : true,
      "monitorSubject" : "authentication.subject.id",
      "securityProfile" : "_default",
      "corsProfile" : "_default"
    },
    "e2fa239a-dc30-4f37-adb0-25c7f87d82a2" : {
      "monitorAPI" : true,
      "monitorSubject" : "authentication.subject.id",
      "securityProfile" : "write:pets"
    },
    "a07a09fc-aa13-4edf-a5e3-5ed6f3d811ca" : {
      "monitorAPI" : true,
      "monitorSubject" : "authentication.subject.id",
      "securityProfile" : "Pass Through"
    }
  }
}
```

**Comment by cwiechmann** [comment]
Thanks for reporting. Too bad, that I haven’t catchend this before releasing 1.6.3. I’ll modify my test-case to capture that bug and fix it.
I’m thinking to release a version 1.6.3-1 as this bug is serious. What do you think @shan916 ?

**Comment by shan916** [comment]
I agree there should be 1.6.3-1 as it is impossible to import an API that was exported by 1.6.3.

**Comment by cwiechmann** [comment]
Fixed with version 1.6.3-1
